src/erp/payment/PaymentProcessor.java

```diff
@@ -0,0 +1,21 @@
+package erp.payment;
+
+import java.util.Scanner;
+
+public class PaymentProcessor {
+
+    public void processPayment() {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Enter payment amount: ");
+        double amount = scanner.nextDouble();
```

Comment:
There's zero error handling for unexpected inputs. If someone dares to enter a string or any non-numeric value, the whole thing will blow up like a bad joke.